### ISSUE TITLE
Add Built-in Services table 

### ISSUE THREAD

**Issue by Rick-Anderson** [aspnet/Docs#9238]
Built-in Services table 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 793395fe-c738-22ce-d0e9-29a850c0fa88
* Version Independent ID: 0546ce3a-3258-9817-0f3f-13281c2f55da
* Content: [Application startup in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup?view=aspnetcore-2.1#services-available-in-startup)
* Content Source: [aspnetcore/fundamentals/startup.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/startup.md)
* Product: **aspnet-core**
* GitHub Login: @ardalis
* Microsoft Alias: **tdykstra**

**Comment by Rick-Anderson** [comment]
per `@danroth27` 	

Agreed - having a table of all built in services would be great. Some of the built-in services are covered in https://docs.asp.net/en/latest/fundamentals/startup.html#services-available-in-startup, but it loosk like IHttpContextAccessor isn't covered anywhere.

**Comment by guardrex** [comment]
Cross-ref: https://docs.microsoft.com/aspnet/core/fundamentals/dependency-injection#framework-provided-services

**Comment by dougbu** [comment]
@guardrex yes, that table exists. But it conflicts with information [about the generic host](https://github.com/aspnet/Docs/blob/master/aspnetcore/fundamentals/host/generic-host.md#default-services) and isn't correct everywhere. What I started on was removing the central table, extending the generic host list, and adding a table for the web host.
